include/deal.II/base/exceptions.h

```diff
@@ -1228,14 +1228,19 @@ namespace StandardExceptions
                                            dealii::ExcCudaError(cudaGetErrorString(error_code)))
 #endif
 
-// A static assert which checks a compile-time condition. This is nothing more
-// than a wrapper around the C++11 <code>static_assert</code> declaration, which
-// we need to disable for non-C++11.
-
-// Just like <code>static_assert</code>, it takes two arguments, a
-// <code>constexpr</code> condition and an error message.
+/**
+ * A static assert which checks a compile-time condition. This is nothing more
```

Comment:
Hm, we had agreed that the macro should simply not be documented. So there was a purpose in not using a doxygen-style comment.